### PR TITLE
Add a quick start which contain some examples.

### PR DESCRIPTION
**Pull request by ni-ze**
Add a quick start which contain some examples.

### CODE REVIEW

File: docs/cn/quick_start.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+## rocketmq-streams 快速搭建
+---
+### 前言
+本文档主要介绍如何基于rocketmq-streams快熟搭建流处理任务。
```
Comment (DongyuanPan) [inline docs/cn/quick_start.md]: This sentense has a typo,  please check 快熟.

File: quick_start.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
+## rocketmq-streams 快速搭建
+---
+### 前言
+本文档主要介绍如何基于rocketmq-streams快熟搭建流处理任务，搭建过程中某些例子会用到rocketmq，可以参考[rocketmq搭建文档见文档](https://rocketmq.apache.org/docs/quick-start/)
```
Comment (DongyuanPan) [inline quick_start.md:4]: This sentense has a typo, please check 快熟.

Comment (RongtongJin) [inline quick_start.md:4]: 可以参考[rocketmq搭建文档]，见文档多余

File: rocketmq-streams-examples/README.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+    }
+}
+
+```
+
+
+### 2、分时间段，统计分组中某字段的和
+
+#### 2.1 安装rocketmq
+可以参考[rocketmq搭建文档见文档](https://rocketmq.apache.org/docs/quick-start/)
```
Comment (RongtongJin) [inline rocketmq-streams-examples/README.md:22]: 同上，“见文档”多余

File: rocketmq-streams-examples/README.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+
+在时间窗口18:02:00 - 18:03:00 内，有1条数据；
+
+分钟统计窗口内，被点击次数最多的网页.
+得到上述数据后，需要按照窗口进行筛选最大值，需要再次计算。
+代码：
+```java
+    public void findMax() {
+        DataStreamSource source = StreamBuilder.dataStream("ns-1", "pl-1");
+        source.fromFile("/Users/nize/result.txt", false)
```
Comment (RongtongJin) [inline rocketmq-streams-examples/README.md:139]: It would be better not to appear personal path.

**Review by duhenglucky — COMMENTED** [review]
缺少了远端checkpoint 的支持

**Comment by ni-ze** [comment]
> 缺少了远端checkpoint 的支持

already supported In the latest code.

File: quick_start.md (hunk 2 above)
Comment (ni-ze) [reply quick_start.md]: Corrected

File: rocketmq-streams-examples/README.md (hunk 3 above)
Comment (ni-ze) [reply rocketmq-streams-examples/README.md]: Corrected

File: rocketmq-streams-examples/README.md (hunk 4 above)
Comment (ni-ze) [reply rocketmq-streams-examples/README.md]: Corrected

File: quick_start.md (hunk 2 above)
Comment (ni-ze) [reply quick_start.md]: Corrected

File: rocketmq-streams-channel-rocketmq/src/main/java/org/apache/rocketmq/streams/source/RocketMQSource.java [hunk 5]
```diff
@@ -41,26 +41,13 @@
...
 
-import java.util.ArrayList;
-import java.util.HashMap;
-import java.util.HashSet;
-import java.util.Iterator;
-import java.util.List;
-import java.util.Map;
-import java.util.Set;
-import java.util.UUID;
+import java.util.*;
```
Comment (duhenglucky) [inline rocketmq-streams-channel-rocketmq/src/main/java/org/apache/rocketmq/streams/source/RocketMQSource.java:49]: _import java.util.*;_ It is not recommended to use * to aggregate all imports.

Comment (ni-ze) [reply rocketmq-streams-channel-rocketmq/src/main/java/org/apache/rocketmq/streams/source/RocketMQSource.java]: all corrected.

File: rocketmq-streams-examples/README.md [hunk 6]
```diff
@@ -0,0 +1,159 @@
...
+
+在时间范围 18:02:00- 18:03:00内：
+
+|userId|点击次数|
+|------|---|
+|   4  | 1 | 
+
+可查看原数据文件，eventTime为时间字段，简单检查后上述结果与预期相符合。
+
+#### 3.2、统计某段时间窗口内，被点击次数最多的网页
```
Comment (duhenglucky) [inline rocketmq-streams-examples/README.md:114]: Could we add chapter 3.3 to describe the scale in/out ability of rocketmq-streams

**Review by duhenglucky — APPROVED** [review]

**Pull request merged**